libs/model/test/launchpad/launchpad.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { Actor, command, dispose } from '@hicommonwealth/core';
+import { BalanceType, commonProtocol } from '@hicommonwealth/shared';
+import chai, { expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { bootstrap_testing, seed } from 'model/src/tester';
+import { afterAll, beforeAll, describe, test } from 'vitest';
+import { ChainNodeAttributes } from '../../src/index';
+import { CreateToken } from '../../src/token';
+
+chai.use(chaiAsPromised);
+
+describe('Launchpad', () => {
+  let actor: Actor;
+  let payload;
+  let community_id: string;
+  let node: ChainNodeAttributes;
+
+  beforeAll(async () => {
+    await bootstrap_testing(true);
+    [node] = (await seed('ChainNode', {
+      url: 'https://base-sepolia-rpc.publicnode.com',
+      private_url: 'https://base-sepolia-rpc.publicnode.com',
```

Comment:
Use Alchemy URLs with `ALCHEMY_PUBLIC_APP_KEY` and `ALCHEMY_PRIVATE_APP_KEY` environment variables:
`https://base-sepolia.g.alchemy.com/v2/${config.ALCHEMY.APP_KEYS.PUBLIC}`.